resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -168,22 +168,124 @@ Specify the connector in the builder:
 #### Setting the max retry time
 
 By default, the Java SDK does not automatically retry any failed requests.
-To enable automatic retries, provide a RetryConfiguration object with the maximum number of seconds to retry requests when building the ApiClient instance.
+To enable automatic retries, provide a RetryConfiguration object with the maximum number of seconds to retry requests and the max number of retries when building the ApiClient instance.
 
 Building a `RetryConfiguration` instance:  
 ```{"language":"java"}
 ApiClient.RetryConfiguration retryConfiguration = new ApiClient.RetryConfiguration();
 retryConfiguration.setMaxRetryTimeSec(30);
+retryConfiguration.setRetryMax(5);
 ```
 
 Setting `RetryConfiguration` instance to `ApiClient`:
 ```{"language":"java"}
         .withRetryConfiguration(retryConfiguration)
 ```
 Set the `maxRetryTimeSec` to the number of seconds to process retries before returning an error.
+Set the 'MaxRetryTimeSec' to the number of seconds to process retries before returning an error.
 When the retry time is a a positive integer, the SDK will follow the recommended backoff logic using the provided configuration.
 The best practices are documented in the [Rate Limiting](https://developer.mypurecloud.com/api/rest/rate_limits.html) Developer Center article.
 
+#### SDK Logging
+
+Logging of API requests and responses can be controlled by a number of parameters on the `Configuration`'s `Logger` instance.
+Need to create an instance of ApiClient.LoggingConfiguration for this SDK.
```

Comment:
Please edit this line and the line above. I didn't mean for you to copy that comment in word for word. Have it show that people need to create an instance of ApiClient.LoggingConfiguration, set values on it and build the APIClient with it.

Reply:
Oh, that is actually shown on lines 207 - 218.  Is that what you were looking for?
![image](https://user-images.githubusercontent.com/80964224/116748169-0b6cc680-a9cd-11eb-8d80-4587170178ae.png)